client/layout/masterbar/logged-in.jsx

```diff
@@ -491,24 +492,20 @@ class MasterbarLoggedIn extends Component {
 	}
 
 	render() {
-		const { isCheckout, isCheckoutPending } = this.props;
+		const { isInEditor, isCheckout, isCheckoutPending } = this.props;
 		const { isMobile } = this.state;
 
+		const currentSegment = 10; //percentage of users that will see the Help Center, not the FAB
```

Comment:
Should this config really live in the `render()` method?

Reply:
Same as above!